llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -181,16 +185,20 @@ static cl::opt<bool> ClWithTls(
     cl::Hidden, cl::init(true));
 
 static cl::opt<bool>
-    CSkipHotCode("hwasan-skip-hot-code",
-                 cl::desc("Do not instument hot functions based on FDO."),
-                 cl::Hidden, cl::init(false));
+    CSelectiveInstrumentation("hwasan-selective-instrumentation",
```

Comment:
CSelectiveInstrumentation -> CSkipHotCode?

Reply:
Becuse the flag controls hwasan-random-skip-rate and hwasan-percentile-cutoff-hot I think it makes more sense to call it hwasan-selective-instrumentation because hwasan-random-skip-rate doesn't use FDO at all.